packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts

```diff
@@ -158,12 +172,27 @@ export function throwOdspNetworkError(
     errorMessage: string,
     statusCode: number,
     response?: Response,
+    responseText?: string,
```

Comment:
I can't find any usages of this... remove it and on the other function?

Reply:
Actually this is added just recently. So may not be yet in telemetry. So lets keep it.